### DISCUSSION TITLE
20/05/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#160] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer domingo NADA
Hoy empezaré con la instalación de los programas y organizar sprint 2.
Tiempo y organización.

**Comment by rpgrca** [comment]
Ayer empecé con la parte del carrito que necesita estar autenticado y mandar la cookie en los requests, me acordé que nunca creé un modelo Cart en el servidor, así que me está complicando la parte del manejo de las respuestas. Hoy voy a seguir con eso.

**Comment by fmdcba** [comment]
Hoy terminé de modificar las pantallas de login, registro y home segun los prototipos de figma. El diseño aun es simple pero es un buen punto de partida para seguir modificándolas. Luego quiero continuar con las pruebas automatizadas de selenium

**Comment by LuciaL152** [comment]
Ayer no hice nada, hoy estuve trabajando todo el día y ahora me puse a ver lo de Android, creo que de impedimento tenemos la falta de organización y de comunicación.
